WakeLockSentinel/WakeLockSentinel_release_method.html

```diff
@@ -0,0 +1,60 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Screen Wake Lock API")}}</div>
+
+<p class="summary">The <strong><code>release()</code></strong> method of the {{domxref("WakeLockSentinel")}} interface releases the {{domxref("WakeLockSentinel")}}, returning a {{jsxref("Promise")}} that is resolved once the sentinel has been successfully released.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<!-- slightly unsure about this because in practice you most likely won't pass it into a var, just call it -->
+<pre class="syntaxbox">var <var>release</var> = WakeLockSentinel.release();</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<p>This method does not receive any parameters.</p>
+
+<h3 id="Returns">Return value</h3>
+
+<p>Returns a {{jsxref("Promise")}} that resolves with <code>undefined</code></p>
+
+<!-- <h3 id="Exceptions">Exceptions</h3> -->
+
+<!-- and I quote "We don't handle failures... it is not like we would turn off the
+screen or so if the lock is released.
+
+All we do is drop our intent for the screen to remain on. The system
+might never turn off the screen (user pref) or there might be a lot of
+other "screen locks" active in the system, like the user playing a
+video at the same time."-->
```

Comment:
```suggestion
```

Reply:
Make a note in the description that in lieu of exceptions, implementor should always listen for a `Release` event. Use the DOMXref() macro to link to it.